src/core/scalebar/qgsscalebarrendererregistry.h

```diff
@@ -0,0 +1,90 @@
+/***************************************************************************
+    QgsScaleBarRendererRegistry.h
+    ---------------------
+    begin                : March 2020
+    copyright            : (C) 2020 by Nyall Dawson
+    email                : nyall dot dawson at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QgsScaleBarRendererRegistry_H
```

Comment:
This needs upper casing (and the header's .h file name above needs lower casing)